src/app/containers/RadioSchedule/Canonical/useData.jsx

```diff
@@ -0,0 +1,31 @@
+import { useState, useEffect } from 'react';
```

Comment:
This is generic, so wouldn't actually live in the radio schedule container. Move to the hooks folder.